lib/workers/repository/update/pr/participants.ts

```diff
@@ -7,10 +7,16 @@ import { sampleSize } from '../../../../util/sample';
 import { codeOwnersForPr } from './code-owners';
 
 async function addCodeOwners(
+  config: RenovateConfig,
   assigneesOrReviewers: string[],
   pr: Pr
 ): Promise<string[]> {
-  return [...new Set(assigneesOrReviewers.concat(await codeOwnersForPr(pr)))];
+  const assignees =
+    config.expandCodeownersGroups && platform.expandGroupMembers
```

Comment:
@rarkins Should we warn if not supported but enabled?

Reply:
Would be useful, updated to warn